pkg/cmd/create/cluster/scenario/create/hub/aws/install_config.yaml

```diff
@@ -10,21 +10,25 @@ controlPlane:
   replicas: {{ if .managedCluster.master }}{{ .managedCluster.master.replicas }}{{ else }}3{{end}}
   platform:
     aws:
+{{ if .managedCluster.aws.master }}
+{{ if .managedCluster.aws.master.rootVolume }}
       rootVolume:
```

Comment:
What happens if rootVolume is not specified and just this in the yaml?  Does it work?

Reply:
as the rootVolume key is in the if it will be not show.

```
compute:
- hyperthreading: Enabled
  name: worker
  platform:
    aws:
      type: m5.xlarge
  replicas: 0
controlPlane:
  hyperthreading: Enabled
  name: master
  platform:
    aws:
      type: m5.xlarge
  replicas: 1
```